docs/articles/deploying-in-aws.md

```diff
@@ -66,69 +66,12 @@ helm install --create-namespace testkube kubeshop/testkube --namespace testkube
 
 After the installation command is complete, you will see the following resources created into your AWS Console.
 
-![AWS Console](../img/aws-resource-console.png)
-
 ![AWS Console2](../img/aws-resource-console-2.png)
 
-![AWS Console3](../img/aws-resource-console-3.png)
-
 Please note that the annotations may vary, depending on your Load Balancer schema type, backend-protocols (you may use http only), target-type, etc. However, this is the bare minimum that should be applied to your configuration.
```

Comment:
This part `(you may use http only)` might be a bit misleading, does it mean if you want to use http or you can use http only?

Reply:
badly phrazed